README.md

```diff
@@ -1 +1,41 @@
-# lboard
\ No newline at end of file
+# lboard
+
+In this project, I have built a web app powered by the Leaderboard APIs. I've achieved by using the Webpack Framework.
+
+## Live Link
+
+[Live Link](https://softwareengineermwi.github.io/lboard/)
```

Comment:
### Please note that this is the final step to complete this project. So, it's so important to leave it professionally and technically completed 🙏 As I see, it's technically perfect 😉 Now let's work on the professional side 🥰

- [ ] Kindly provide a valid live link to your app or if you're managing to deploy it later, replace the text with "Coming soon" or something like that 😄